requirements.txt

```diff
@@ -0,0 +1 @@
+-r requirements/heroku.txt
```

Comment:
Yep, I think that's a good idea. That's how it's done in the ``wagtail-template`` repo: https://github.com/torchbox/wagtail-template/blob/master/requirements.txt